src/Middleware/HeaderPropagation/src/DependencyInjection/HeaderPropagationExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.HeaderPropagation
+{
+    public static class HeaderPropagationExtensions
```

Comment:
We'll want docs on all new public API (once design is finished).

Reply:
Still looking for docs here. Something basic like `Provides extension methods for using header propagation.` would suffice.

Reply:
I can do this, but it doesn't look like a common practice across the repo.